python/lsst/pipe/analysis/fakesAnalysis.py

```diff
@@ -494,11 +496,11 @@ def plotWithTwoHists(xs, ys, xName, yName, xLabel, yLabel, title, plotInfoDict,
 
     Notes
     -----
-    The `plotInfoDict` needs to be a dict containing camera, filter, visit,
-    tract and dataset (jointcal or not), it is used to add information to the
-    plot. Returns the median and sigma MAD for the x and y values.
+    ``plotInfoDict`` needs to be a `dict` containing keys "camera", "filter",
+    "visit", "tract" and "photoCalibDataset" (e.g. "jointcal" or "fgcm"), and
```

Comment:
Should keys be in between double-backticks in cases like this? (Keeping the `"` in there, too). I am actually not sure, and could not find the answer in the DevGuide...

Reply:
Yeah, I took some time pondering this one too.  I think, since keys are strings, at minimum the double quotes are appropriate when explicitly naming them.  They are both used inside double-backticks here: https://developer.lsst.io/python/numpydoc.html#dictionary-types
However, this is just a note (and I'm not entirely convinced of the value of documenting the keys here in the Notes section in addition to in the Parameters list, and in all the other notes in all the other functions...soooo many that could get out of sync...), so I think it's not too crucial to include the multiple layers of ticks here (I'd be more inclined to remove the Notes sections all together, but did not want to question the original author's motivation, so went with the minimal update!)